tools/install.py

```diff
@@ -19,7 +19,8 @@ def abspath(*args):
   return os.path.abspath(path)
 
 def load_config():
-  s = open('config.gypi').read()
+  with open('config.gypi') as f:
+    s = f.read()
```

Comment:
```suggestion
    return ast.literal_eval(f.read())
```

Reply:
Added in a fixup commit. Thanks for the review!